README.md

```diff
@@ -321,11 +324,19 @@ in   | cmd.meter_ext.get_report      | null       |                         | Re
 
 #### Interface props
 
-Name       | Value example | Description
------------|---------------|-------------
-`delta_t`  |               | time delta
-`prv_data` |               | previous meter reading
-`unit`     |               |
+Name         | Value example | Description
+-------------|---------------|-------------
+`delta_t`    |               | time delta
+`prv_data`   |               | previous meter reading
+`unit`       |               | One of sup_units. For meter_unknown it will be a number.
+`rate_type`  | "export"      | Defines whether reading is based on consumption (import) or production (export).
+`meter_type` | 10            | Number representing meter type. Applicable ONLY for meter_unknown service.
```

Comment:
Do you have any more information on what meter types this would be?

Reply:
deleted this property, because it will be handled using separate service within separate Pull Request :)